Editor/Window/RegisterComputeInput.cs

```diff
@@ -109,6 +111,11 @@ private async void OnRegisterComputeButtonClicked()
                     _registerComputeStatusBox.Show(StatusBox.StatusBoxType.Error, Strings.AnywherePageStatusBoxDefaultErrorText, registerResponse.ErrorMessage, url, Strings.ViewLogsStatusBoxUrlTextButton);
                 }
             }
+            else if (_computeState is ComputeStatus.Registered)
+            {
+                await _stateManager.ComputeManager.RegisterFleetCompute(_computeName,
+                    _stateManager.AnywhereFleetId, _stateManager.AnywhereFleetLocation, _ipAddress);
```

Comment:
If the compute is already registered, why do we need to register it again? We should only need to update the UI with appropriate Compute Name, and status, right?